rero_mef/cli.py

```diff
@@ -1000,11 +759,10 @@ def save(output_file_name, name, from_date, until_date, quiet, enqueue):
     :param quiet: Surpress output.
```

Comment:
I know this is not in your PR, but…

```suggestion
    :param quiet: Suppress output.
```